stepper/src/fce.rs

```diff
@@ -43,7 +43,7 @@ pub fn main() {
 }
 
 #[fce]
-pub fn invoke(init_peer_id: String, aqua: String, prev_data: String, data: String) -> StepperOutcome {
+pub fn invoke(init_peer_id: String, aqua: String, prev_data: Vec<u8>, data: Vec<u8>) -> StepperOutcome {
```

Comment:
Maybe change major version of fce, fluence-faas and aquamarine-vm due to this change?

Reply:
Done